examples/mp-metrics-histogram/README_pt.adoc

```diff
@@ -44,10 +43,9 @@ $ curl -X GET http://localhost:8080/mp-metrics-histogram/weather/histogram
 
 == Usando `@Histogram`
 
-As métricas do MicroProfile tem uma função que permite criar um histogram de dados.
+As métricas do MicroProfile tem uma função que permite criar um histograma de dados.
 
-Para utilizar esta função, injetamos um objeto `MetricRegistry`, registrar o histogram e 
-Para utilizar esta función, injecta un objeto `MetricRegistry`, registra el Histograma, e adiciona dados ao histograma, como mostrado abaixo.
+Para utilizar esta função, injetamos um objeto `MetricRegistry`, registramos um Histogram e adicionamos dados ao histograma como mostrado abaixo:
```

Comment:
@Daniel-Dos,  only one small thing here.

Change registramos um Histogram to registramos um `Histogram`

Reply:
done . thanks.